### DISCUSSION TITLE
header element doesn't save position

### DISCUSSION THREAD

**Discussion by nobaraos12** [JulianPrieber/littlelink-custom#250] [Issues]
header element on the demo site doesn't save the position when dragged up the row.

**Comment by JulianPrieber** [comment]
Can't reproduce, I'm not having any issue with it 🤔

**Comment by nobaraos12** [comment]
> Can't reproduce, I'm not having any issue with it thinking
Upon refresh it gets pushed to the bottom, I'm using firefox
![Screenshot from 2022-12-03 14-48-00-obfuscated](https://user-images.githubusercontent.com/118019611/205459212-8ef038fe-5388-4c55-85de-197499a54454.png)

**Comment by JulianPrieber** [comment]
Weird, switching browsers I still can't get anything near this behavior.

Do you get a console error when trying to move your block?

**Comment by nobaraos12** [comment]
oddly enough this issue seemed to resolve itself, although if it happens again ill send in the console error

**Comment by JulianPrieber** [comment]
👍

**Comment by JulianPrieber** [comment]
If you encounter this problem again, you can continue your post here.
